### PR TITLE
Add example of Ruby version manager for developer setup documentation

### PR DESCRIPTION
**Pull request by gbraad**
Add an example of `rbenv` to install an updated version of Ruby

See #111

### CODE REVIEW

**Comment by Fryguy** [comment]
@bdunne Thoughts?

File: developer_setup.md [hunk 1]
```diff
@@ -88,6 +88,26 @@
   * [rbenv](https://github.com/sstephenson/rbenv)
 * Required Ruby version is 2.2.2+
 
+#### Example using rbenv
+
+First get the requirements to perform a build of a newer Ruby version
+
+```bash
+sudo yum install -y git-core zlib zlib-devel gcc-c++ patch readline readline-devel libyaml-devel libffi-devel openssl-devel make bzip2 autoconf automake libtool bison curl sqlite-devel
```
Comment (bdunne) [inline developer_setup.md]: I think that packages belong up in the OS specific section above with an appropriate comment

**Comment by bdunne** [comment]
@gbraad Aside from the packages, I don't see anything that isn't already covered in the `rbenv` link a couple lines above.  I'd prefer not to add documentation for other projects here unless we really need to.  Did you run into a problem that isn't covered under the `rbenv` install guide?

**Comment by kbrock** [comment]
I'd like to agree.
Making sure all the packages are up in the OS section sounds good.
Duplicating the `rbenv` docs seems a bit much.

**Pull request closed without merging**